### PR TITLE
Update documentation for places to receive InputEventShortcut

### PR DESCRIPTION
**Pull request by anniryynanen**
I didn't test it myself, but https://docs.godotengine.org/en/latest/tutorials/inputs/inputevent.html#how-does-it-work and https://docs.godotengine.org/en/latest/classes/class_node.html#class-node-private-method-shortcut-input say so.

### CODE REVIEW

File: doc/classes/InputEventShortcut.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 		Represents a triggered keyboard [Shortcut].
 	</brief_description>
 	<description>
-		InputEventShortcut is a special event that can be received in [method Node._unhandled_key_input]. It is typically sent by the editor's Command Palette to trigger actions, but can also be sent manually using [method Viewport.push_input].
+		InputEventShortcut is a special event that can be received in [method Node._shortcut_input] and [method Node._unhandled_key_input]. It is typically sent by the editor's Command Palette to trigger actions, but can also be sent manually using [method Viewport.push_input].
```
Comment (Sauermann) [inline doc/classes/InputEventShortcut.xml:7]: Adding `_shortcut_input` is correct, but the preexisting text isn't.

So you should either name the function, that is primarily intended for shortcuts:
```suggestion
		InputEventShortcut is a special event that can be received in [method Node._shortcut_input]. It is typically sent by the editor's Command Palette to trigger actions, but can also be sent manually using [method Viewport.push_input].
```

or include all functions, that shortcuts can be received in:
```suggestion
		InputEventShortcut is a special event that can be received in [method Node._input], [method Node._shortcut_input] and [method Node._unhandled_input]. It is typically sent by the editor's Command Palette to trigger actions, but can also be sent manually using [method Viewport.push_input].
```

File: doc/classes/InputEventShortcut.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 		Represents a triggered keyboard [Shortcut].
 	</brief_description>
 	<description>
-		InputEventShortcut is a special event that can be received in [method Node._unhandled_key_input]. It is typically sent by the editor's Command Palette to trigger actions, but can also be sent manually using [method Viewport.push_input].
+		InputEventShortcut is a special event that can be received in [method Node._input], [method Node._shortcut_input] and [method Node._unhandled_input]. It is typically sent by the editor's Command Palette to trigger actions, but can also be sent manually using [method Viewport.push_input].
```
Comment (AThousandShips) [inline doc/classes/InputEventShortcut.xml:7]:
```suggestion
		InputEventShortcut is a special event that can be received in [method Node._input], [method Node._shortcut_input], and [method Node._unhandled_input]. It is typically sent by the editor's Command Palette to trigger actions, but can also be sent manually using [method Viewport.push_input].
```
Please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).

**Comment by Mickeon** [comment]
Commit and title could say something different. They read as if this is a bug being reported, instead of a documentation fix.

**Review by Sauermann — APPROVED** [review]
The changes fix the previously outdated statement.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**